### ISSUE TITLE
zone.getFirstPropertyValue(...) is null

### ISSUE THREAD

**Issue by technicalperson** [jobisoft/TbSync#68]
## Your environment

TbSync version: 0.7.7
Thunderbird version: 52.7.0 (64-bit)
OS: Ubuntu 16.04

[X] Yes, I have installed the latest available version from 
https://github.com/jobisoft/TbSync/releases 
and my issue is not yet fixed, I can still reproduce it.

## Expected behavior
Synchronization with Office 365 account runs without errors.

## Actual behavior
Seeing TbSync: Error in status bar. Error console contents:
zone.getFirstPropertyValue(...) is null  tbsync.jsm:1324
	tbSync.getTimezoneInfoObject chrome://tbsync/content/tbsync.jsm:1324:32
	tbSync.getTimezoneInfo chrome://tbsync/content/tbsync.jsm:1283:22
	eas.sync.Calendar.getWbxmlFromThunderbirdItem chrome://tbsync/content/provider/eas/calendarsync.js:203:88
	eas.sync.sendLocalChanges< chrome://tbsync/content/provider/eas/sync.js:135:66
	InterpretGeneratorResume self-hosted:1213:8
	next self-hosted:1120:9
	TaskImpl_run resource://gre/modules/Task.jsm:319:42
	bound TaskImpl_run self-hosted:957:17

## Steps to reproduce
Connect to Office365 account. In Calendar tab of Thunderbird click "Synchronize" 

The debug.log was sent via e-mail (used the title of this issue as subject of the email).

**Comment by jobisoft** [comment]
I just released a new [beta](https://github.com/jobisoft/TbSync/releases/download/v0.7.7/TbSync-beta.xpi). I do not think it will solve your problem entirely, but will put the timezone object, which causes the error into the log, so you do not need to export your calendar data anymore.

Please note: By installing this version, you are switching to the [beta release channel](https://github.com/jobisoft/TbSync/wiki/What-is-the-beta-release-channel%3F)

**Comment by jobisoft** [comment]
I consider this fixed. Reopen if otherwise

**Comment by nitro322** [comment]
I encountered this problem today.  Updated the 0.7.7.4 through the link on the TbSync help tab, tried syncing again, and it worked like a champ.  Beautiful.  :-)

Just FYI, after re-syncing I got this in the debug log:

** Tue Apr 24 2018 16:39:21 GMT-0500 (CDT) **
[JavaScript Error: "Couldn't find (UTC-06:00) Central Time (US & Canada)" {file: "file:///usr/lib64/thunderbird/extensions/%7Be2fda1a4-762b-4020-b5ad-a41df1933103%7D/components/calTimezoneService.js" line: 199}]

I don't know if that's helpful to you or not, but wanted to share just in case.
